Source/Resources/Content/MSGraph/DeviceManagement/GroupPolicyConfigurations/OS - System Sleep Timeouts.json

```diff
@@ -45,5 +27,5 @@
       ]
     }
   ],
-  "description": "Sets the system sleep timeout for devices to never when plugged in and to 20 minutes when on battery."
-}
\ No newline at end of file
+  "description": "What is it: Sets the system sleep timeout for devices to never when plugged in."
```

Comment:
@david-simeon this is only sleep config that we will keep. Did we have an updated comment for this one?

Reply:
@metamenos I thought so but I guess not...

Reply:
**What does this do?**
Sets the system sleep timeout for devices to 'Never' when plugged in.

**Why should you use this?**
If you want to prevent desktop computers from going to sleep so you can connect remotely at any time.

**What is the end-user impact?**
Users' devices will not go to sleep when plugged in.

**Learn more**
N/A

@metamenos is this ok?